XCUITests/NavigationTest.swift

```diff
@@ -135,24 +130,6 @@ class NavigationTest: BaseTestCase {
     private func checkFirefoxSyncScreenShown() {
         // Disable check, page load issues on iOS13.3 sims, issue #5937
         waitForExistence(app.webViews.firstMatch, timeout: 20)
-        // Workaround BB iOS13
```

Comment:
Removing workaround for BuddyBuild since that was commented and not used anymore